crates/consensus/src/transaction/mod.rs

```diff
@@ -55,7 +55,7 @@ pub mod serde_bincode_compat {
 /// Represents a minimal EVM transaction.
 #[doc(alias = "Tx")]
 #[auto_impl::auto_impl(&, Arc)]
-pub trait Transaction: fmt::Debug + any::Any + Send + Sync + 'static {
+pub trait Transaction: fmt::Debug + any::Any + Send + Sync + 'static + Typed2718 {
```

Comment:
let's put Typed2718 first

```suggestion
pub trait Transaction: Typed2718 + fmt::Debug + any::Any + Send + Sync + 'static {
```